dask_cloudprovider/cloudprovider.yaml

```diff
@@ -48,19 +48,36 @@ cloudprovider:
     filesystem_size: 40 # Default root filesystem size for scheduler and worker VMs in GB
 
   azure:
-    experiment_name: "dask-cloudprovider" # default name of the Experiment to submit
-    initial_node_count: 1 # Initial node count
-    jupyter: False # by default start the Jupyter lab instance on the headnode
-    jupyter_port: 9000 # default Jupyter lab port on the headnode
-    dashboard_port: 9001 # default Dask dashboard port on the headnode
-    scheduler_port: 9002 # default for the scheduler port if using from outside of cluster VNET
-    scheduler_idle_timeout: 1200 # default timeout for dask-scheduler process to shutdown if idle (in seconds)
-    worker_death_timeout: 30 # default timeout for dask-worker process upon losing scheduler
-    additional_ports: [] # list of tuples of additional ports to map/forward
-    admin_username: "" # username to log in to the AzureML Training Cluster for 'local' runs
-    admin_ssh_key: "" # password to log in to the AzureML Training Cluster for 'local' runs
-    telemetry_opt_out: False # by default we log the version of the AzureMLCluster being used, set to True to opt-out
-    datastores: [] # default list of datastores to mount
+    location: "eastus2" # The Azure location to launch your cluster
+    resource_group: null # The Azure resource group for the cluster
```

Comment:
what will default resource group name be?

Reply:
It defaults to `None` which will raise a `ConfigError`. We expect the user to set this, either as a kwarg or in their Dask config.

The same goes for the vnet and security group.